OsmAnd/src/net/osmand/plus/keyevent/KeyEventHelper.java

```diff
@@ -141,6 +141,12 @@ private QuickAction findAction(int keyCode) {
 		return device != null ? device.findAction(keyCode) : null;
 	}
 
+	private boolean isLetterForbid(int keyCode){
+		boolean letterAllowedScreens = mapActivity.isMapVisible() || mapActivity.getMapRouteInfoMenu().isVisible();
```

Comment:
Do we need to check here for AndroidUtils.isActivityNotDestroyed(this) or settings.MAP_ACTIVITY_ENABLED? probably better introduce here several local variables & make a robust check